### PR TITLE
swift4 support

### PR DESCRIPTION
**Pull request by Jiar**

### CODE REVIEW

File: Example/Segmentio.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -481,19 +486,22 @@
...
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CODE_SIGN_STYLE = Manual;
+				DEVELOPMENT_TEAM = "";
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
 					"$(PROJECT_DIR)",
 				);
 				INFOPLIST_FILE = Segmentio/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
```
Comment (Tykhonkov) [inline Example/Segmentio.xcodeproj/project.pbxproj]: is it required changes for swift 4?

**Comment by Tykhonkov** [comment]
@Jiar  thanks for PR, we are going to roll it out when Xcode 9 will be public

**Comment by grifas** [comment]
Xcode 9 is now public. Can you merge this PR ?

**Comment by andre991** [comment]
@Tykhonkov Please you can merge this PR? Thanks.

**Comment by DantePuglisi** [comment]
Is `IPHONEOS_DEPLOYMENT_TARGET` required to be 11.0? iOS 7 and above will deploy Swift 4 so it doesn't seem necessary to change that.

**Comment by andre991** [comment]
@DantePuglisi  Ok, but you can explain this?
captura de tela 2017-10-21 as 11 38 23

File: Example/Segmentio.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Jiar) [reply Example/Segmentio.xcodeproj/project.pbxproj]: NO,It just my mistake. I have resubmitted it.

**Comment by Jiar** [comment]
Sorry, I take a mistake. I have resubmitted It.

**Comment by andre991** [comment]
Thanks @Jiar

**Comment by grifas** [comment]
So could you please merge this PR ? Thanks

**Comment by npalamar** [comment]
Hi guys! Could your merge this MR?

**Comment by AndrewPetrov** [comment]
@Jiar Thank you for attention to our component. Unfortunately I can't accept your request because of a lot of code, that can be generated only with "pod update" and Xcode auto correction. But this changes is already in the repo and you can get fresh Segmentio version soon. Let us know if anything isn’t ok and feel free to reopen this PR

**Pull request closed without merging**